components/de_amp/browser/de_amp_util.cc

```diff
@@ -23,8 +23,8 @@ constexpr char kGetHtmlTagPattern[] = "(<\\s*?html\\s.*?>)";
 // To see the expected behaviour of this regex, please see unit tests in
 // de_amp_util_unittest.cc
 constexpr char kDetectAmpPattern[] =
-    "(?:<.*?\\s.*?(amp|⚡|⚡=\"\\s*\"|⚡=\'\\s*\'|amp=\"\\s*\"|amp='\\s*')(?:\\s.*"
-    "?>|>|/>))";
+    "(?:<.*?\\s.*?(amp|⚡|⚡=\"(?:true|\\s*)\"|⚡=\'(?:true|\\s*)\'|amp=\"(?:true|"
```

Comment:
The `(?:` just makes it a non-capturing group, because we only care about detecting that it's AMP, not in the actual content itself.